src/appsco-customers-page.html

```diff
@@ -0,0 +1,540 @@
+<link rel="import" href="../bower_components/polymer/polymer.html">
```

Comment:
Remove all unused imports, CSS, HTML, methods and properties since this should be an empty page. It should contain empty 'content' and 'info' sections only with general properties and methods. As we implement other components required imports, HTML, CSS and JS will be created.